git/cmd.py

```diff
@@ -13,6 +13,8 @@
 import errno
 import mmap
 
+from collections import OrderedDict
```

Comment:
@Byron I agree it's worth fixing, especially since `git.odict` is still there.

Should I also look into adding (back) testing for Py2.6 in `.travis.yml`, indicating that the tests for Py2.6 are allowed to fail? I feel that may help in at least *knowing* that Py2.6 compatibility is broken.